tests/test_lint.py

```diff
@@ -1,12 +1,20 @@
 import os
 import subprocess
+from math import nan
 from unittest.mock import patch
 
 import libsbml
 import pandas as pd
-import petab
 import pytest
+
+import petab
 from petab import (lint, sbml)  # noqa: E402
+from petab.C import *
+
+# import fixtures
+pytest_plugins = [
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constant name "pytest_plugins" doesn't conform to UPPER_CASE naming style](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4859825)